src/content/getusermedia/getdisplaymedia/js/main.js

```diff
@@ -38,9 +47,14 @@ function errorMsg(msg, error) {
   }
 }
 
-const startButton = document.getElementById('startButton');
```

Comment:
Why did you remove this line?

Reply:
not removed, moved up. We do declare the the UI-related accessors at the top